src/Mvc/Mvc.Cors/test/CorsApplicationModelProviderTest.cs

```diff
@@ -19,52 +19,6 @@ public class CorsApplicationModelProviderTest
 {
     private readonly IOptions<MvcOptions> OptionsWithoutEndpointRouting = Options.Create(new MvcOptions { EnableEndpointRouting = false });
 
-    [Fact]
-    public void OnProvidersExecuting_SetsEndpointMetadata_IfCorsAttributeIsPresentOnController()
```

Comment:
I assume we have some end-to-end tests covering this scenario. Do you know where they are?

Reply:
Here:
https://github.com/dotnet/aspnetcore/blob/9af007591a271f176cc74f374d2c239908f1ac7d/src/Mvc/test/Mvc.FunctionalTests/CorsTestsBase.cs#L11

https://github.com/dotnet/aspnetcore/blob/9af007591a271f176cc74f374d2c239908f1ac7d/src/Mvc/test/Mvc.FunctionalTests/CorsEndpointRoutingTests.cs#L6